internal/backends/qt/qt_accessible.rs

```diff
@@ -0,0 +1,439 @@
+                                accessible_item: this,
+                            });
+                        let property_tracker = Box::pin(property_tracker);
+                        Box::into_raw(unsafe { core::pin::Pin::into_inner_unchecked(property_tracker) })
+                    });
+            armFocusTracker();
+        }
+
+        ~Slint_accessible_item() {
+            rust!(Slint_accessible_item_dtor [
+                m_rustItem: *mut ItemWeak as "void*",
+                m_focusTracker: *mut i_slint_core::properties::PropertyTracker<HasFocusPropertyTracker> as "void*",
+                m_stateTracker: *mut i_slint_core::properties::PropertyTracker<AccessibleItemPropertiesTracker> as "void*"] {
+                if !m_rustItem.is_null() {
+                    Box::from_raw(m_rustItem);
+                };
+                Box::from_raw(m_stateTracker);
+                Box::from_raw(m_focusTracker);
+            });
+        }
+
+        void armFocusTracker() {
+            rust!(Slint_accessible_item_arm_focus_tracker [
+                m_rustItem: &ItemWeak as "void*",
+                m_focusTracker: &i_slint_core::properties::PropertyTracker<HasFocusPropertyTracker> as "void*"] {
+                    let item = m_rustItem.clone();
+                    let m_focusTracker = unsafe { Pin::new_unchecked(m_focusTracker) };
+                    m_focusTracker.evaluate_as_dependency_root(move || {
+                        if let Some(item_rc) = item.upgrade() {
+                            item_rc.accessible_string_property(i_slint_core::accessibility::AccessibleStringProperty::Has_focus);
+                        }
+                    });
+                    unsafe { Pin::into_inner_unchecked(m_focusTracker) };
+            });
+        }
+
+        void *rustItem() const override {
+            return m_rustItem;
+        }
+
+        void *takeRustItem() {
+            auto item = m_rustItem;
+            m_rustItem = nullptr;
+            return item;
+        }
+
+        QWindow *window() const override {
+            return parent()->window();
+        }
+
+        // properties and state
+        QString text(QAccessible::Text t) const override {
+            return item_string_property(m_rustItem, t);
+        }
+
+        QAccessible::State state() const override {
+            QAccessible::State state;
+            state.active = 1;
+            state.focusable = 1;
+            state.focused = (item_string_property(m_rustItem, QAccessible::UserText) == "true") ? 1 : 0;
+            return state; /* FIXME */
+        }
+
+    private:
+        void *m_rustItem = nullptr; // A rust item to make accessible (Actually a ItemWeak!)
+        void *m_stateTracker = nullptr; // (Actually a Pin<Box<PropertyTracker<AccessibleItemPropertiesTracker>>>)
+        void *m_focusTracker = nullptr; // (Actually a Pin<Box<PropertyTracker<AccessibleItemPropertiesTracker>>>)
```

Comment:
What i'd do here is a single pointer being a `Pin<Box<SlintAccessibleItem>>`
where SlintAccessibleItem is a rust struct containing all the tracker.  So you only have one box instead 3, and this is easier to handle in the rust code